pontoon/insights/templates/insights/widgets/insights.html

```diff
@@ -0,0 +1,172 @@
+                            </li>
+                        </ul>
+                    </div>
+                    <ul class="period-selector noselect clearfix">
+                        <li><div class="active selector" data-period="12">12m</div></li>
+                        <li><div class="selector" data-period="6">6m</div></li>
+                        <li><div class="selector" data-period="3">3m</div></li>
+                        <li><div class="selector" data-period="1">1m</div></li>
+                    </ul>
+                </h3>
+                <div id="managers" class="active-users">
+                    <canvas class="chart" height="110" width="110"></canvas>
+                    <div class="active-wrapper">
+                        <span class="active noselect"></span>
+                    </div>
+                    <span class="total noselect"></span>
+                    <h4>Active managers</h4>
+                </div>
+                <div id="reviewers" class="active-users">
+                    <canvas class="chart" height="110" width="110"></canvas>
+                    <div class="active-wrapper">
+                        <span class="active noselect"></span>
+                    </div>
+                    <span class="total noselect"></span>
+                    <h4>Active reviewers</h4>
+                </div>
+                <div id="contributors" class="active-users">
+                    <canvas class="chart" height="110" width="110"></canvas>
+                    <div class="active-wrapper">
+                        <span class="active noselect"></span>
+                    </div>
+                    <span class="total noselect"></span>
+                    <h4>Active contributors</h4>
+                </div>
+            </div>
+        </section>
+        <section class="half">
+            <div class="block clearfix">
+                <h3 class="controls">
+                    Unreviewed suggestions lifespan
+                    <div class="fa fa-info selector"></div>
+                    <div class="tooltip">
+                        How much time it takes on average to review a suggestion within the selected time frame.
+                    </div>
+                </h3>
+                <canvas
+                    id="unreviewed-suggestions-lifespan-chart"
+                    data-lifespans="{{ data.unreviewed_lifespans }}"
+                    width="410"
+                    height="160">
+                </canvas>
+            </div>
+        </section>
+    </section>
+
+    <section class="translation-activity">
+        <figure class="block">
+            <h3 class="controls">
+                Translation activity
+                <div class="fa fa-info selector"></div>
+                <div class="tooltip">
+                    <p>Impact of adding translations and source strings on the overal translation completion.</p>
+                    <ul>
+                        <li class="completion">
```

Comment:
[nitpick] What do you think about using macros for the columns like that? They could reduce the size of this html.

Reply:
Good point! I should add macros for tooltips, and probably also for the three active users "pie" charts.